stylix/palette.nix

```diff
@@ -81,11 +112,15 @@ in
         # and not anything indirect such as filling a template, otherwise
         # the output of the palette generator will not be protected from
         # garbage collection.
-        default = pkgs.runCommand "palette.json" { } ''
-          ${lib.getExe cfg.paletteGenerator} \
-            "${cfg.polarity}" \
-            ${lib.escapeShellArg "${cfg.image}"} \
-            "$out"
+        default = pkgs.runCommand "raw-palette.json" { } ''
+          ${lib.getExe pkgs.matugen} \
```

Comment:
Nit: I prefer to use `nativeBuildInputs` to add the package to the builder's PATH rather than interpolating the exe path directly into code